src/apps/altcp_tls/altcp_tls_mbedtls.c

```diff
@@ -89,6 +89,9 @@
 #ifndef ALTCP_MBEDTLS_ENTROPY_LEN
 #define ALTCP_MBEDTLS_ENTROPY_LEN   0
 #endif
+#ifndef ALTCP_MBEDTLS_RNG_FN
+#define ALTCP_MBEDTLS_RNG_FN   mbedtls_entropy_func
```

Comment:
Shouldn't this be `mbedtls_entropy_update_manual` instead? `mbedtls_entropy_func` extracts randomness

Reply:
Ah, the supplied function is for later extraction. Nevermind